Author: Daniel Leidert (dale) <daniel.leidert@wgdd.de>
Description: Fixes broken element declarations by adding the tag omission characters
for SGML and enable SGML features
(closes: <URL:http://bugs.debian.org/256119>).

<URL:http://packages.debian.org/changelogs/pool/main/d/docbook/current/changelog#versionversion4.4-1>
--- a/docbook-4.3/dbpoolx.mod
+++ b/docbook-4.3/dbpoolx.mod
@@ -633,6 +633,9 @@
 
 <!-- xml:base: base URI -->
 
+<![%sgml.features;[
+<!ENTITY % xml-base.attrib "">
+]]>
 <!ENTITY % xml-base.attrib
 	"xml:base	CDATA		#IMPLIED">
 
--- a/docbook-4.3/docbookx.dtd
+++ b/docbook-4.3/docbookx.dtd
@@ -59,7 +59,7 @@
 <!-- ...................................................................... -->
 <!-- Enable SGML features ................................................. -->
 
-<!ENTITY % sgml.features "IGNORE">
+<!ENTITY % sgml.features "INCLUDE">
 <![%sgml.features;[
 <!ENTITY % xml.features "IGNORE">
 ]]>
--- a/docbook-4.3/htmltblx.mod
+++ b/docbook-4.3/htmltblx.mod
@@ -63,6 +63,9 @@
   >
 
 <!-- Does not contain lang or dir because they are in %common.attribs -->
+<![%sgml.features;[
+<!ENTITY % i18n "">
+]]>
 <!ENTITY % i18n
  "xml:lang    NMTOKEN        #IMPLIED"
   >
@@ -92,11 +95,11 @@
   "valign     (top|middle|bottom|baseline) #IMPLIED"
   >
 
-<!ELEMENT colgroup (col)*>
-<!ELEMENT col      EMPTY>
-<!ELEMENT tr       (th|td)+>
-<!ELEMENT th       (%para.char.mix; | %tabentry.mix;)*>
-<!ELEMENT td       (%para.char.mix; | %tabentry.mix;)*>
+<!ELEMENT colgroup %ho; (col)*>
+<!ELEMENT col      %ho; EMPTY>
+<!ELEMENT tr       %ho; (th|td)+>
+<!ELEMENT th       %ho; (%para.char.mix; | %tabentry.mix;)*>
+<!ELEMENT td       %ho; (%para.char.mix; | %tabentry.mix;)*>
 
 <!ATTLIST colgroup
   %attrs;
@@ -206,7 +209,7 @@
     summary     CDATA          #IMPLIED
     width       CDATA        #IMPLIED
     border      CDATA        #IMPLIED
-    rules       (none | groups | rows | cols | all)      #IMPLIED
+    rules       CDATA        #IMPLIED
     cellspacing CDATA        #IMPLIED
     cellpadding CDATA        #IMPLIED
     align       (left|center|right)   #IMPLIED
--- a/docbook-4.4/docbookx.dtd
+++ b/docbook-4.4/docbookx.dtd
@@ -61,7 +61,7 @@
 <!-- ...................................................................... -->
 <!-- Enable SGML features ................................................. -->
 
-<!ENTITY % sgml.features "IGNORE">
+<!ENTITY % sgml.features "INCLUDE">
 <![%sgml.features;[
 <!ENTITY % xml.features "IGNORE">
 ]]>
--- a/docbook-4.4/htmltblx.mod
+++ b/docbook-4.4/htmltblx.mod
@@ -63,6 +63,9 @@
   >
 
 <!-- Does not contain lang or dir because they are in %common.attribs -->
+<![%sgml.features;[
+<!ENTITY % i18n "">
+]]>
 <!ENTITY % i18n
  "xml:lang    NMTOKEN        #IMPLIED"
   >
@@ -92,11 +95,11 @@
   "valign     (top|middle|bottom|baseline) #IMPLIED"
   >
 
-<!ELEMENT colgroup (col)*>
-<!ELEMENT col      EMPTY>
-<!ELEMENT tr       (th|td)+>
-<!ELEMENT th       (%para.char.mix; | %tabentry.mix; | table | informaltable)*>
-<!ELEMENT td       (%para.char.mix; | %tabentry.mix; | table | informaltable)*>
+<!ELEMENT colgroup %ho; (col)*>
+<!ELEMENT col      %ho; EMPTY>
+<!ELEMENT tr       %ho; (th|td)+>
+<!ELEMENT th       %ho; (%para.char.mix; | %tabentry.mix; | table | informaltable)*>
+<!ELEMENT td       %ho; (%para.char.mix; | %tabentry.mix; | table | informaltable)*>
 
 <!ATTLIST colgroup
   %attrs;
